bin/init.sh

```diff
@@ -17,3 +17,7 @@ DIR="$( cd -P "$( dirname "$SOURCE" )" && pwd )"
 
 cd $DIR/../build
 npm install
+cd ..
+cp -a build build-1
+cp -a build build-2
+cp -a build build-3
```

Comment:
Should be a little more obvious now that I've checked in all the files! Wasn't thinking straight at 2:00am I guess :smiley: